src/css/styles.scss

```diff
@@ -51,9 +51,7 @@
     image-rendering: -moz-crisp-edges; // FF
 
     z-index: var(--zIndex-canvas);
-  }
 
-  #canvas {
```

Comment:
we still need to distinguish the main canvas from any other canvas because we don't want the `position: absolute` to e.g. target the canvas in image export dialog

Reply:
good point , now I remember breaking it earlier 🤦🏻

Reply:
haha, I thought it was me :)

Reply:
I broke it when introducing position absolute and you fixed it :)